components/Address.json

```diff
@@ -0,0 +1,53 @@
+{
+  "Address": {
+    "name": "address",
+    "schema": {
+      "$ref": "./Address.json#/address"
+    }
+  },
+  "Addresses": {
+    "name": "addresses",
+    "schema": {"$ref": "./Address.json#/addresses"}
+  },
+  "orNull": {
+    "name": "address or null",
+    "schema": {
+      "$ref": "./Address.json#/addressOrNull"
+    }
+  },
+  "Miner": {
```

Comment:
No need for `Miner`, it's just an `address`.

Reply:
It's specified because it is explained as the miner address

Reply:
What is the advantage of this vs. titling it this and using `address` as the type?

Reply:
I talked to Shane some more yesterday and now I understand fully the difference between content descriptors and schemas. I've decided that we should always define content descriptors in place (e.g. in the method params like I've done in the current impl). This means we'll need to get rid of these aliases like `Miner`